src/io/flutter/bazel/Workspace.java

```diff
@@ -220,6 +220,8 @@ static Workspace loadUncached(@NotNull Project project) {
     final String readonlyPath = "../READONLY/" + root.getName();
     final VirtualFile readonlyRoot = root.findFileByRelativePath(readonlyPath);
     VirtualFile configFile = root.findFileByRelativePath(PLUGIN_CONFIG_PATH);
+    if (configFile == null && readonlyRoot == null) return null;
```

Comment:
why not short circuit if the configFile == null OR readonlyRoot == null?
That should indicate we are in an environment where you can't actually run a g3 app.

Reply:
I'm not clear on how the readonlyRoot works. I thought that if `configFile` is null but `readOnlyRoot` exists, then we can still run a g3 app? Since the code below finds a config file within readonly.

Maybe I should instead check for only `configFile` later, e.g.:
```
    VirtualFile configFile = root.findFileByRelativePath(PLUGIN_CONFIG_PATH);
    if (configFile == null && readonlyRoot != null) {
      configFile = readonlyRoot.findFileByRelativePath(PLUGIN_CONFIG_PATH);
    }
    if (configFile == null) return null;
```